time/unit.go

```diff
@@ -56,6 +56,20 @@ func (tu Unit) Value() (time.Duration, error) {
 	return 0, errUnrecognizedTimeUnit
 }
 
+// Count returns the number of units contained within the duration.
+func (tu Unit) Count(d time.Duration) int {
+	if d < 0 {
+		return -1
+	}
+
+	if dur, found := unitsToDuration[tu]; found {
+		return int(d / dur)
```

Comment:
no need to downcast here, return `int64`.

Reply:
To be honest, I feel like `int` is the more appropriate type to return. Most functions in the standard library return an `int` as opposed to an `int64` so I feel that it would be more idiomatic.